libcaf_net/test/net-test.hpp

```diff
@@ -105,6 +106,63 @@ class mock_stream_transport : public caf::net::octet_stream::lower_layer {
   caf::net::multiplexer* mpx_;
 };
 
+class mock_web_socket_app : public caf::net::web_socket::upper_layer::server {
```

Comment:
Since this is a re-usable class now, a bit of docu would be nice. :)

```
/// Implements a trivial WebSocket application that stores the contents of all received text frames in ...
class mock_web_socket_app : public caf::net::web_socket::upper_layer::server {
```